test/test-ncu.js

```diff
@@ -41,7 +41,7 @@ describe('npm-check-updates', function () {
         });
 
         it('should output only upgraded with --jsonUpgraded', function() {
-            return spawn('node', ['bin/npm-check-updates', '--jsonUpgraded'], '{ "dependencies": { "express": "1" } }')
+            return spawn('node', ['bin/ncu', '--jsonUpgraded'], '{ "dependencies": { "express": "1" } }')
```

Comment:
had to change; else, `JSON.parse` fails because of the extra `[INFO]` output that gets printed (since role of `bins` have [swapped](https://github.com/tjunnone/npm-check-updates/pull/203/files#r51649427)